VideoLocker/src/main/java/org/edx/mobile/module/notification/EdxParsePushBroadcastReceiver.java

```diff
@@ -32,37 +32,23 @@
 public class EdxParsePushBroadcastReceiver extends ParsePushBroadcastReceiver {
     protected final Logger logger = new Logger(getClass().getName());
 
+    private boolean enableFlyinWindow = false;
```

Comment:
I see. So if we're not using the fly in window yet, should we leave a TODO here to enable this pre Android L when it's ready?